src/DIRAC/WorkloadManagementSystem/Client/JobStatus.py

```diff
@@ -92,9 +95,10 @@ def __init__(self, state):
             RESCHEDULED: State(6, [WAITING, RECEIVED, DELETED, FAILED], defState=RESCHEDULED),
             MATCHED: State(5, [RUNNING, FAILED, RESCHEDULED, KILLED], defState=MATCHED),
             WAITING: State(4, [MATCHED, RESCHEDULED, DELETED], defState=WAITING),
-            STAGING: State(3, [CHECKING, WAITING, FAILED, KILLED], defState=STAGING),
-            CHECKING: State(2, [STAGING, WAITING, RESCHEDULED, FAILED, DELETED], defState=CHECKING),
-            RECEIVED: State(1, [CHECKING, WAITING, FAILED, DELETED], defState=RECEIVED),
+            STAGING: State(3, [WAITING, FAILED, KILLED], defState=STAGING),
```

Comment:
@fstagni at least give people that still care about v7r3 a chance to review!
The CHECKING must not be dropped here
https://github.com/DIRACGrid/DIRAC/commit/4d3e4777780d0e665c73a40cea4d3fc3cb2f1ee8

Reply:
oops I missed that one! 

@michmx can you create a new PR amending this (and correct https://github.com/DIRACGrid/DIRAC/pull/7121 ?)

Reply:
Sorry for that, I copy-pasted from our own patch in the server! I will fix it ASAP.

Reply:
Thanks!